src/main/kotlin/xyz/atrius/waystones/Waystones.kt

```diff
@@ -41,6 +43,8 @@ class Waystones : KotlinPlugin() {
 //        )
         registerNamespaces(
             CommandNamespace("waystones").register(
```

Comment:
lets add `ws` as an alias for the namespace

Reply:
this is already being handled by bukkit/spigot for us within the plugin.yml and i've just confirmed it does work

Reply:
I see, I guess that is fine for now then.